pytest_instafail.py

```diff
@@ -46,7 +46,8 @@ def pytest_collectreport(self, report):
         # Show errors occurred during the collection instantly.
         TerminalReporter.pytest_collectreport(self, report)
         if report.failed:
-            self.rewrite("")  # erase the "collecting" message
+            if self.isatty:
```

Comment:
Yep, that's better! I don't really see a reason to drop it, if it's still so simple to keep the compatibility.